image-carousel.html

```diff
@@ -225,6 +225,16 @@
             value: 1000
           },
 
+          /**
+           * If true, a `selection-changed` event will be fired when the selection
+           * changes, whether via autoscroll, or user initiation by pressing a
+           * dot or left/right buttons
+           */
```

Comment:
Instead of writing 
> event will be fired when the selection changes
can we have something else like `when displayed image changes`. I'm saying this as word `selection` can be confusing as user may not be doing any selection.